presto-main/src/main/java/io/prestosql/execution/QueryStateMachine.java

```diff
@@ -455,14 +455,14 @@ private QueryStats getQueryStats(Optional<StageInfo> rootStage)
                 blockedReasons.addAll(stageStats.getBlockedReasons());
             }
 
-            PlanFragment plan = stageInfo.getPlan();
-            if (plan != null && plan.getPartitionedSourceNodes().stream().anyMatch(TableScanNode.class::isInstance)) {
-                physicalInputDataSize += stageStats.getPhysicalInputDataSize().toBytes();
-                physicalInputPositions += stageStats.getPhysicalInputPositions();
+            physicalInputDataSize += stageStats.getPhysicalInputDataSize().toBytes();
+            physicalInputPositions += stageStats.getPhysicalInputPositions();
 
-                internalNetworkInputDataSize += stageStats.getInternalNetworkInputDataSize().toBytes();
-                internalNetworkInputPositions += stageStats.getInternalNetworkInputPositions();
+            internalNetworkInputDataSize += stageStats.getInternalNetworkInputDataSize().toBytes();
+            internalNetworkInputPositions += stageStats.getInternalNetworkInputPositions();
 
+            PlanFragment plan = stageInfo.getPlan();
+            if (plan != null && plan.getPartitionedSourceNodes().stream().anyMatch(TableScanNode.class::isInstance)) {
                 rawInputDataSize += stageStats.getRawInputDataSize().toBytes();
```

Comment:
should we depreciate/remove raw stats on stage, query level? @dain ?

Reply:
FWIU your new stats are better in every way, so I'd just remove them.

Reply:
This will be done in subsequent PRs.

There is also question what to do with `processedInput*` on `QueryStats` level. It's also accumulated only for stages that contain `TableScanNode.class`. We could:
* accumulate it for all stages (also intermittent stages with exchanges only)
* split it into `internalNetworkProcessedInputDataSize` and `physicalProcessedInputDataSize`
* remove it
* leave it as is

I think we could split it so it can be used to measure compression ratio for network/table scans separately